pyccel/codegen/printing/ccode.py

```diff
@@ -113,6 +114,14 @@ def _print_While(self,expr):
         code = code + "\n %s" % self._print(expr.body) + "\n}"
         return (code)
 
+    def _print_PyArg_ParseTupleNode(self, expr):
```

Comment:
We should have access to this function from CWrapper._print_FunctionDef even if it is defined in CWraapperCodePrinter. And that way we can't accidentally use it while writing normal C

Reply:
yeah thats true i just put it in ccode for the time being (test purposes only)